dask_cuda/tests/test_spill.py

```diff
@@ -149,3 +162,136 @@ def get_data(worker, total_size):
                     assert dc > 0
                 else:
                     assert dc == 0
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        {
+            "device_memory_limit": 1e9,
+            "memory_limit": 1e9,
+            "host_target": 0.6,
+            "host_spill": 0.7,
+            "spills_to_disk": False,
+        },
+        {
+            "device_memory_limit": 250e6,
+            "memory_limit": 250e6,
```

Comment:
This number for host memory limit is quite low.  Just importing a few libraries like Pandas and a few other libraries can get us close to this point.

Reply:
In particular I suspect that this is responsible for some of the CI errors